components/tooltip/script.js

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import components from 'coherent-gameface-components';
+import template from './template.html';
+const TOOLTIP_MARGIN = 10;
+
+class Tooltip extends HTMLElement {
+    constructor() {
+        super();
+        this.template = template;
+
+        this.state = {
+            visible: false
+        };
+    }
+
+    connectedCallback() {
+        this.position = this.getAttribute('position') || 'top';
+        this.showOn = this.getAttribute('on') || 'click';
+        this.hideOn = this.getAttribute('off') || 'click';
+        this.elementSelector = this.getAttribute('for');
+        // TODO: check if this.elementSelector is valid and attached to DOM

```

Comment:
![shia-la-beouf-just-do-it](https://user-images.githubusercontent.com/21058220/143006475-18d26842-d22b-45ff-9908-4027a9fcfe8a.gif)

Reply:
Added check if the element exists and if it doesn't - show an error and return.